File: src/vehicle/rental/system/vehicleType/CargoVan.java
package vehicle.rental.system.vehicleType;

import vehicle.rental.system.rental.Vehicle;

import java.math.BigDecimal;
import java.math.RoundingMode;

public class CargoVan extends Vehicle {

    private final int driverExperience;

    public CargoVan(String brand, String model, BigDecimal value, int driverExperience) {
        super(brand, model, value, "cargoVan");
        this.driverExperience = driverExperience;
    }

    public int getDriverExperience() {
        return driverExperience;
    }

    public void displayInfo(){
        System.out.println("\nA cargo van valued at $" + getValue().setScale(2, RoundingMode.DOWN) + " and the driver has "  + driverExperience + "  years of experience\n");
    }
}

File: src/VehicleType/CargoVan.cs
using VehicleRentalSystem.Rental;

namespace VehicleRentalSystem.VehicleType;

public class CargoVan : Vehicle
{
    public int DriverExperience { get; }

    public CargoVan(string brand, string model, decimal value, int driverExperience)
        : base(brand, model, value, "cargoVan")
    {
        DriverExperience = driverExperience;
    }

    public void DisplayInfo()
    {
        Console.WriteLine($"\nA cargo van valued at ${Value:F2} and the driver has {DriverExperience} years of experience\n");
    }
}
